spec/workers/attachments/extract_fulltext_job_spec.rb

```diff
@@ -91,39 +102,47 @@
     end
   end
 
-  shared_examples "only file name indexed" do
-    it "updates the attachment's DB record with file_tsv" do
-      expect(extracted_attributes["fulltext"]).to be_blank
-      expect(extracted_attributes["fulltext_tsv"]).to be_blank
-      expect(extracted_attributes["file_tsv"].size).to be > 0
-    end
-  end
-
   context "with the file not readable" do
     before do
       allow(attachment).to receive(:readable?).and_return(false)
     end
 
-    include_examples "only file name indexed"
+    it "updates the attachment's DB record with file_tsv" do
+      perform_enqueued_jobs
+      expect(extracted_attributes["fulltext"]).to be_blank
+      expect(extracted_attributes["fulltext_tsv"]).to be_blank
+      expect(extracted_attributes["file_tsv"].size).to be > 0
+    end
   end
 
-  context "with exception in extraction" do
+  context "with exception during extraction" do
     let(:exception_message) { "boom-internal-error" }
     let(:logger) { Rails.logger }
 
     before do
-      allow_any_instance_of(Plaintext::Resolver).to receive(:text).and_raise(exception_message)
+      allow_any_instance_of(Plaintext::Resolver).to receive(:text).and_raise(exception_message) # rubocop:disable RSpec/AnyInstance
 
       allow(logger).to receive(:error)
 
       allow(attachment).to receive(:readable?).and_return(true)
     end
 
-    it "logs the error" do
-      extracted_attributes
+    it "raises the exception to update Job's internal state" do
+      expect { perform_enqueued_jobs }.to raise_error(exception_message)
+    end
+
+    it "logs the exception" do
+      perform_enqueued_jobs
+    rescue StandardError
       expect(logger).to have_received(:error).with(/boom-internal-error/)
```

Comment:
Note: This test (and the test below) will always pass if there is no exception raised, regardless of an error message being logged.

It's not too bad, because we validate the raising of an exception in another test case, but that's why I suggested the `perform_enqueued_jobs rescue nil` syntax before.

Reply:
You're right. I've fixed it again and added a rubocop config file for `spec/` dir to ignore [Style/RescueModifier](https://docs.rubocop.org/rubocop/cops_style.html#stylerescuemodifier).